node/src/command.rs

```diff
@@ -23,21 +23,24 @@ use crate::{
     service::{new_partial, CalamariRuntimeExecutor, DolphinRuntimeExecutor, MantaRuntimeExecutor},
 };
 use codec::Encode;
-use cumulus_client_service::genesis::generate_genesis_block;
+use cumulus_client_cli::generate_genesis_block;
 use cumulus_primitives_core::ParaId;
 use frame_benchmarking_cli::{BenchmarkCmd, SUBSTRATE_REFERENCE_HARDWARE};
 use log::info;
 use manta_primitives::types::Header;
-use polkadot_parachain::primitives::AccountIdConversion;
```

Comment:
Is not `polkadot_parachain` being used?

Reply:
Yes we use some of the primitives in `polkadot_parachain::primitives`